apps/studio/components/interfaces/DataWarehouse/CreateWarehouseCollection.tsx

```diff
@@ -103,6 +127,21 @@ export const CreateWarehouseCollectionModal = () => {
                   </FormItemLayout>
                 )}
               />
+              <FormField_Shadcn_
+                control={form.control}
+                name="retention_days"
+                render={({ field }) => (
+                  <FormItemLayout
+                    label="Retention days"
+                    layout="horizontal"
```

Comment:
thinking of using labelOptional for the upsell - something like this, but wondering wdyt as well though @saltcod 

![image](https://github.com/user-attachments/assets/f8a29a59-0505-427d-a779-c027fd7b26d1)

Reply:
hmm actually i realised - we don't allow reconfiguring the retention period after the collection is created right?

Reply:
what about this though - wdyt?
- Changing the layout to vertical
- labelOptional to highlight that you can't change retention
- Upsell in an Admonition instead

![image](https://github.com/user-attachments/assets/fe922a8d-5aee-4add-a91a-f9b9fc659aa5)

Reply:
@joshenlim checked with chase and we plan to allow updating retention for a collection. I think it's a good idea to upsell anyway if they need more than the free tier allows.

Reply:
+++ to the upgrade modal there. I think customers will know what they want right there and then, let them upgrade in place if they want to 👍